package.json

```diff
@@ -21,6 +21,7 @@
     "migrate": "ts-node node_modules/.bin/node-pg-migrate -j ts",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
+    "vendor-secp256k1": "mkdir -p src/vendored/@noble/secp256k1 && cp node_modules/@noble/secp256k1/index.ts src/vendored/@noble/secp256k1/index.ts",
```

Comment:
This library is esm-only, which jest is unable to handle. Luckily we can simply copy the single ts source file of this library into our repo and it is compiled to cjs. This a hacky workaround, and the approach is unlikely to work with most other other esm-only dependencies. The long term solution is to move away from jest.